metrics_sdk/lib/opentelemetry/sdk/metrics/state/metric_stream.rb

```diff
@@ -45,6 +45,7 @@ def collect(start_time, end_time)
                 @meter_provider.resource,
                 @instrumentation_scope,
                 @aggregation.collect(start_time, end_time),
+                @aggregation.instance_variable_get(:@aggregation_temporality),
```

Comment:
Are we using instance variable get because we cannot expose this with an attribute reader?